reco_utils/dataset/wikidata.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import requests
+
+def find_wikidataID(name):
+    """Find the entity ID in wikidata from a title string.
+    Args:
+        name (str): A string with search terms (eg. "Batman (1989) film")
+    Returns:
+        entityID: wikidata entityID corresponding to the title string. 
+                  'entityNotFound' will be returned if no page is found
+    """
+    r = requests.get("https://en.wikipedia.org/w/api.php?action=query&list=search&srsearch="+name+"&format=json&prop=pageprops&ppprop=wikibase_item")
+    try:
+        pageID = r.json()["query"]["search"][0]["pageid"]
+        r = requests.get("https://en.wikipedia.org/w/api.php?action=query&prop=pageprops&format=json&pageids="+str(pageID))
+        try:
+            entity_id = r.json()["query"]["pages"][str(pageID)]["pageprops"]["wikibase_item"]
+        except:
+            entity_id = "entityNotFound"
+    except:
```

Comment:
I see that you are returning entity not found in some cases. You might want to name the except instead of using the generic value. See more info here: https://stackoverflow.com/a/16511493/5620182.

Reply:
Int his commit I changed the exceptions to catch requests exceptions and used .get for the dictionaries: 51b0fd2